nmdc_server/schemas_submission.py

```diff
@@ -110,6 +110,8 @@ class SubmissionMetadataSchema(SubmissionMetadataSchemaCreate):
     created: datetime
     status: str
     author: schemas.User
+    templates: List[str]
+    study_name: str
```

Comment:
Should these be optional? I checked out your branch locally and when loading the submission list I get:

```
backend-1  | pydantic.error_wrappers.ValidationError: 1 validation error for MetadataSubmissionResponse
backend-1  | response -> results -> 5 -> study_name
```

There are ~50 submissions in dev that have no `studyName` (I haven't checked prod). I think this can happen if the submission never makes it past the study form.

Reply:
fixed in f37cf212e3faa6934db6ac8e9c80d78dd8bb98da